### PR TITLE
ID:FPCO-7857; DONE: 80; HOURS: 12;add null check

### PR DESCRIPTION
**Pull request by ekata123**

### CODE REVIEW

File: src/lib/Theme.ts [hunk 1]
```diff
@@ -1006,9 +1006,12 @@ export default class Theme {
                 path.join(Theme.BUILD_FOLDER, `${assetHash}_themeBundle.common.js`),
                 'utf-8'
             );
-
+            let customTemplates;
             const themeBundle = evaluateModule(bundleFiles);
-            const customTemplates = themeBundle.getCustomTemplates();
+            if(themeBundle && themeBundle.getCustomTemplates())
+            {
```
Comment (ZaidRehman) [inline src/lib/Theme.ts]: Formatting issue

File: src/lib/Theme.ts [hunk 2]
```diff
@@ -1006,9 +1006,12 @@ export default class Theme {
                 path.join(Theme.BUILD_FOLDER, `${assetHash}_themeBundle.common.js`),
                 'utf-8'
             );
-
+            let customTemplates;
```
Comment (ZaidRehman) [inline src/lib/Theme.ts]: Add default value, and also add comments

File: src/lib/Theme.ts [hunk 3]
```diff
@@ -1006,9 +1006,11 @@ export default class Theme {
                 path.join(Theme.BUILD_FOLDER, `${assetHash}_themeBundle.common.js`),
                 'utf-8'
             );
-
+            let customTemplates=[]
             const themeBundle = evaluateModule(bundleFiles);
-            const customTemplates = themeBundle.getCustomTemplates();
+            if(themeBundle && themeBundle.getCustomTemplates()){
```
Comment (vivek8690) [inline src/lib/Theme.ts]: if `getCustomTemplates` is not defined inside the theme it will break again @ekata123

**Review by vivek8690 — CHANGES_REQUESTED** [review]
Check comment

File: src/lib/Theme.ts (hunk 3 above)
Comment (ekata123) [reply src/lib/Theme.ts]: @vivek8690 resolved comment

**Review by vivek8690 — APPROVED** [review]

**Pull request merged**